### PR TITLE
Clarify and improve wording  in Express quickstart

### PR DESCRIPTION
**Pull request by nbarbettini**
I noticed some grammar issues in the Express quickstart. Also clarified that if you download the sample app, you should follow those instructions (since they differ from this quickstart).

### CODE REVIEW

File: _source/quickstart-fragments/nodejs/express-auth-code.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ exampleDescription: Express.js Auth Code Example
 
 ## Okta Node.js/Express.js Quickstart
 
-To handle the authentication callback from Okta, your Express server will need to handle this callback.  In this section we'll show you how to install and configure the [@okta/oidc-middleware](https://github.com/okta/okta-oidc-js/tree/master/packages/oidc-middleware) library, referred to below as ExpressOIDC, it will add the needed callback handler to your application.
+Your Express server will need to handle the authentication callback from Okta.  In this section we'll show you how to install and configure the [@okta/oidc-middleware](https://github.com/okta/okta-oidc-js/tree/master/packages/oidc-middleware) library, referred to below as ExpressOIDC, to do this automatically.
 
-> If you would prefer to download a complete sample application, please visit [Express Sample Applications for Okta][].
+> If you would prefer to download a complete sample application, please visit [Express Sample Applications for Okta][] and follow the instructions in the readme.
```
Comment (jmelberg-okta) [inline _source/quickstart-fragments/nodejs/express-auth-code.md]: We link out to the Angular, React, and Vue samples as well. Mind updating that text while we're at it? I fear we'll forget to update it in the future.

**Comment by nbarbettini** [comment]
@jmelberg-okta In reference to your comment - I found and updated the link in the React quickstart, but didn't find any for Angular and Vue. Are you thinking of a link somewhere else?

**Comment by nbarbettini** [comment]
Updated and rebased. I'll merge this in a day or two unless anyone has more comments :)

**Comment by jmelberg-okta** [comment]
Thanks @nbarbettini! Odd that they weren't showing up via search. Could be that they were added only 5 days ago. They should have showed up after rebasing. You can find them ~10 lines down here:
- [Angular](https://raw.githubusercontent.com/okta/okta.github.io/source/_source/quickstart-fragments/angular/default-example.md)
- [Vue](https://raw.githubusercontent.com/okta/okta.github.io/source/_source/quickstart-fragments/vue/default-example.md)

**Comment by nbarbettini** [comment]
Ah! I thought I rebased first, but I must have done it second. Time for some sleep. 💤

File: _source/quickstart-fragments/vue/default-example.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ This guide will walk you through integrating authentication into a Vue app with
 
 At the end of the Vue instructions you can choose your server type to learn more about post-authentication workflows, such as verifying tokens that your Vue application can send to your server.
 
-> If you would prefer to download a complete sample application, please visit [Vue Sample Applications for Okta][].
+> If you would prefer to download a complete sample application, please visit [Vue Sample Applications for Okta][] and follow the instructions in that readme.
```
Comment (robertjd) [inline _source/quickstart-fragments/vue/default-example.md]: We have other callouts in other langs that should get this same change.

**Comment by nbarbettini** [comment]
As requested by @robertjd, I found any occurrence of "sample app*" in our quickstarts and replaced it with this standard text:

> If you would prefer to download a complete sample application instead, please visit [link] and follow those instructions.

**Pull request merged**